src/sankoff_matrix.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019-2022 bito project contributors.

```

Comment:
@ognian- could you have a look over this file?

Reply:
I note that if we template on state_count we can use the fixed-size Eigen matrices, which have advantages for efficiency.